seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/sink/KafkaSinkWriter.java

```diff
@@ -136,7 +136,7 @@ private Properties getKafkaProperties(Config pluginConfig) {
 
     // todo: parse the target field from config
     private SeaTunnelRowSerializer<byte[], byte[]> getSerializer(Config pluginConfig, SeaTunnelRowType seaTunnelRowType) {
-        return new DefaultSeaTunnelRowSerializer(pluginConfig.getString("topics"), seaTunnelRowType);
+        return new DefaultSeaTunnelRowSerializer(pluginConfig.getString("topic"), seaTunnelRowType);
```

Comment:
Change it to use static String `TOPIC` maybe better.

Reply:
Thans,i will modify it.
https://github.com/apache/incubator-seatunnel/pull/2863 This pr does not conflict with the current pr, because this is the doc of the connector V1 version.

Reply:
Does the community have the code style required? I use the code style format that comes with IDEA, which cannot pass the CI pipeline.Thank you for your help.

Reply:
check this: https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/setup.md#install-jetbrains-idea-checkstyle-idea-plugin

Reply:
I checked the code format and submitted it again, please reopen the CI pipeline.

Reply:
![pipe](https://user-images.githubusercontent.com/45089228/192191445-a9fb2d4b-af25-4e4b-a912-adf38a58eaf4.png)

Is this a CI pipeline problem?

Reply:
Network problem, I will rerun it.

Reply:
Thanks.